easybuild/easyconfigs/g/GROMACS/README.md

```diff
@@ -104,11 +108,19 @@ engines.
     of enabling support for the `cray-python` modules. Therefore the corresponding 
     EasyConfigs of GROMACS have also been replaced.**
 
+### GROMACS 2023.2 with AMD GPU support for CPE 22.12
+
+  * There are two easyconfig files for running GROMACS with AMD GPU acceleration (ROCm support via hipSYCL), that follows [installation guide](https://manual.gromacs.org/2023.2/install-guide/index.html#sycl-gpu-acceleration-for-amd-gpus):
+
+    * MPI version for multi-node use, with [HeFFTe](https://manual.gromacs.org/2023.2/install-guide/index.html#using-heffte) supporting offload to multiple GPUs 
+    ([GPU aware MPI](https://manual.gromacs.org/2023.2/install-guide/index.html#gpu-aware-mpi-support) enabled, support for PME decomposition)
+    * Both MPI and thread-MPI version with [VkFFT](https://manual.gromacs.org/2023.2/install-guide/index.html#using-vkfft) recommended for a single GPU use (no support for a direct GPU communication, without PME decomposition)
```

Comment:
> no support for a direct GPU communication

MPI version has support for direct GPU communication, which is why it's good for single-node multi-GPU runs (as long as PME decomposition is not required; then you need HeFFTe). It is only thread-MPI which does not support direct GPU comms.

Reply:
Thanks for clarification, I was not entirely sure. Should have asked.

Reply:
The second point could still use some refinement, I think.

There are two choices discussed here:
* MPI or threadMPI and
* VkFFT, rocFFT or HeFFTe+rocFFT.

I see little benefit in using thread-MPI on LUMI, so perhaps things can be simpler if this is described separately in the following way: the thread-MPI build can be convenient for single-node only use, but since it does not support direct GPU communication (with the SYCL backend) from a performance point of view it is only practical for single-GPU runs, but even in this case it has no benefit over the MPI build.

This allows discussing the FFT option independently:
  * rocFFT: for PME decomposition (to run the PME task across multiple GPUs) HeFFT is required which itself requires the rocFFT backend on AMD;
  * VkFFT is (currently) faster but as it can not be used as a HeFFTe backend, it is not compatible with PME decomposition. Hence, for optimal performance VkFFT is recommended for single GPU simulations (standalone or ensemble) or in multi-GPU runs with exactly one separate PME rank (i.e. `-npme 1`).

Reply:
This is getting into the details of GROMACS implementation, features and usage, but for these readme notes to be useful, they need to be correct (as well as concise). Adding a lot more information here can pose some maintenance risk, so we should avoid that, but most of the above requirements/limitations are not expected to change in the coming major release(es).

Perhaps linking to the GROMACS manual may be helpful for reader of this readme, e.g.:
https://manual.gromacs.org/documentation/current/install-guide/index.html#using-heffte
https://manual.gromacs.org/documentation/current/install-guide/index.html#using-vkfft